robottelo/hosts.py

```diff
@@ -1739,7 +1739,7 @@ def cli(self):
         self._cli._configured = True
         return self._cli
 
-    def install_satellite_or_capsule_package(self):
+    def install_satellite_or_capsule_package(self, install_package=True):
```

Comment:
If installing the package is optional then we should modify the name of this function as its made/sounds to install the package only.

Reply:
I was thinking the same but I'm having a hard time coming up with something new.

Reply:
got any suggestions?

Reply:
how about `optional_satellite_or_capsule_package_install`

Reply:
updated the pr. I split the helper into `install_satellite_or_capsule_package` and `enable_satellite_or_capsule_module`

Reply:
PRT has passed last time so I'm not retriggering it again as the change was not destructive.